### PR TITLE
[13.0][MIG] partner_contact_birthdate

### PR DESCRIPTION
**Pull request by newtratip**
Ready for review.

### CODE REVIEW

**Review by CarlosRoca13 — APPROVED** [review]
LGTM :+1:

**Comment by newtratip** [comment]
@CarlosRoca13 Thank you for reviewed :)

**Comment by newtratip** [comment]
@kittiu @ps-tubtim @Saran440 Can you review this ?

File: partner_contact_birthdate/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Copyright 2014-2015  Grupo ESOC <www.grupoesoc.es>
```
Comment (NL66278) [inline partner_contact_birthdate/__init__.py]: Remove copyright lines from init files.

File: partner_contact_birthdate/views/res_partner.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+     Copyright 2015 Grupo ESOC Ingeniería de Servicios, S.L.U. -->
+
+<odoo>
+    <record id="view_personal_information_birthdate_date" model="ir.ui.view">
```
Comment (NL66278) [inline partner_contact_birthdate/views/res_partner.xml]: The convention is to use the same id as in the inherited from module, but give a distinct name. Now the partner_personal_information_page did not follow this convention, so you have a choice of using personal_information here, or the original id. I would prefer the original id view_partner_form.

**Review by NL66278 — CHANGES_REQUESTED** [review]
:+1: Looks good in general, but a few nags. See other comments.

File: partner_contact_birthdate/__init__.py (hunk 1 above)
Comment (newtratip) [reply partner_contact_birthdate/__init__.py]: @NL66278 
Please suggest me that why init files don't have copyright lines but other files such as res_partner.py have copyright lines.

Thank you.

File: partner_contact_birthdate/views/res_partner.xml (hunk 2 above)
Comment (newtratip) [reply partner_contact_birthdate/views/res_partner.xml]: Good idea.

Comment (newtratip) [reply partner_contact_birthdate/views/res_partner.xml]: @NL66278 
Edited id to original id

Thank you for your review.

File: partner_contact_birthdate/__init__.py (hunk 1 above)
Comment (NL66278) [reply partner_contact_birthdate/__init__.py]: Actually the contributing guidelines are not speaking anymore about this. But the logic is that there is not really anything to claim copyright for...

Comment (newtratip) [reply partner_contact_birthdate/__init__.py]: @NL66278 
I got it. This is new for me.

Thank you for reviewed :)

**Comment by Saran440** [comment]
@newtratip please, rebase this pr to 2 commit -> isort and migrate.

**Review by NL66278 — APPROVED** [review]
:+1: Satisfied now. Just squash the last three commits into one.

**Comment by newtratip** [comment]
@Saran440 and @NL66278 
I squash the last three commit into one already.

Thank you both for your suggest :)

**Pull request merged**